<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OmgFieldsPageBase</class>
 <widget class="QWizardPage" name="OmgFieldsPageBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Specify occurrence layer fields</string>
  </property>
  <property name="subTitle">
   <string>This page is only used if you elected to use a QGIS layer for the source of your occurrence data. If the layer is a non-spatial layer (without point geometry) you need to specify the X and Y fields.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X field</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cboXField"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y field</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cboYField"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Species label field</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cboTaxonField"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Ocurrence frequency field</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cboFrequencyField"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
